mods/ts/rules/nod-structures.yaml

```diff
@@ -25,6 +25,7 @@ NAPOWR:
 		Range: 4c0
 		MaxHeightDelta: 3
 	WithIdleOverlay@LIGHTS:
+		RequiresCondition: !disabled
```

Comment:
I have touched the structures using `RequiresPower`, but `RequiresPower` will have to grant a condition before it can be used for the other structures that must maintain some level of functionality.